### PR TITLE
Use new `PolyInitialized` qualifier

### PR DESCRIPTION
**Pull request by Ao-senXiong**

### CODE REVIEW

**Comment by wmdietl** [comment]
I've realized that this repository contains a copy of `checker-qual.jar`. So when we add and use a new qualifier, we need to update that file. I did that in https://github.com/eisop/jdk/pull/39/commits/2f2fcd89f41f798b03b3e16a32d16172e184454a .

**Comment by wmdietl** [comment]
> @wmdietl It looks like there are still the same failed checks. Could you tell me how to update check-qual.jar file? I will take a look and try to fix it.

When I look at `checker-qual.jar` from https://github.com/eisop/jdk/commit/2f2fcd89f41f798b03b3e16a32d16172e184454a I correctly see the `PolyInitialized.class` file, so I don't think this is the issue.

You can simply do `./gradlew assemble` in the corresponding CF branch and then copy the `checker-qual.jar` file over. That's what I did above, so something else must still be amiss.

**Comment by Ao-senXiong** [comment]
I think the qualifier could be found after I add the polyinitialized in java/base folder. However, we need a jdk17u fork in eisop org to fix this problem now.

**Review by wmdietl — APPROVED** [review]
#42 removed the extra .jar file and now all tests pass.
This can be merged once the main PR is ready.

**Comment by Ao-senXiong** [comment]
Thanks!

File: src/java.base/share/classes/org/checkerframework/checker/initialization/qual/PolyInitialized.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * A polymorphic qualifier for the freedom-before-commitment initialization tracking type-system.
+ *
+ * <p>Any method written using {@link PolyInitialized} conceptually has three versions: one in which
```
Comment (wmdietl) [inline src/java.base/share/classes/org/checkerframework/checker/initialization/qual/PolyInitialized.java:14]: @Ao-senXiong Please sync this file with the original in ECF.

**Review by wmdietl — CHANGES_REQUESTED** [review]
Looks good, just the javadoc for PolyInitialized needs to be updated here to be in sync.

**Review by wmdietl — APPROVED** [review]
I copied over the file.

**Comment by Ao-senXiong** [comment]
> I copied over the file.

Thanks!

**Pull request merged**